README.rst

```diff
@@ -0,0 +1,120 @@
+.. image:: https://cdn.crate.io/web/2.0/img/crate-avatar_100x100.png
+    :width: 100px
+    :height: 100px
+    :alt: Crate.IO
+    :target: https://crate.io
+
+==================
+Crate Java Testing
+==================
+
+The ``crate-java-testing`` project contains a test layer which can be used
```

Comment:
explain what a testlayer is, maybe also state that it extends from/is an ExternalResource